### ISSUE TITLE
listLimit not supported?

### ISSUE THREAD

**Issue by CDRO** [fsaris/news_ttnewsimport#40]
Hello there,

I figured that listLimit should work (according to documentation and source code in `TtNewsPluginMigrat.php`) but it turns out, the migration information is missing in `flexform.txt`.

Do you have an example on how this should be integrated in there?

Best Regards
Tizian

**Comment by derhansen** [comment]
Yes, this seems to be wrong. In `ext:tt_news` the setting `listLimit` in flexform will limit the amount of records shown in the pagination. This is not a limit for the total amount of records to display as the plugin migration does.

Quick fix:

```
case 'listLimit':
    $this->addFieldToArray($new, $value, 'settings.list.paginate.itemsPerPage', 'additional');
    break;
```

**Comment by CDRO** [comment]
Cool, thanks for the quick fix, can we expect to see this in a bear future release?
